frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -89,6 +91,8 @@ local symbol_prefix = {
         mem_usage = "",
         wifi_status = "",
         wifi_status_off = "",
+        page_turning_inverted = "PgT:I",
+        page_turning_regular = "PgT:R",
```

Comment:
No matter how you like your "PgT", I don't think this can be the sole 4 letters among the "icons" where everything else is an icon and none uses letters.

Reply:
I thought you had said use them everywhere

Reply:
I was just answering you telling you could use PgT somewhere and PT somewhere else, I didn't had my eyes and memories on the various letters/icons/compact existing stuff.
The existing code is more worth than my words :)

Reply:
So what do you think of this now then https://github.com/koreader/koreader/pull/12249#discussion_r1700713703 with your eyes and memories refreshed.

Reply:
> okay how about this, (because I really, really like it) PgT and PgT:I as icons (I know, I know), the arrows as compact, and then letters could be PT or whatever...

Well, as I said, whether you like it or not, "icons" mode should have... icons, not the letter-based and long `PgT:I` (!).
And compact should be ... compact, so arrows would work.
"letters" could be the only one to have PgT: - even if I prefer shorter. But as it's not the one you use, it does not really need to have PgT :)